source/faucet/main.d

```diff
@@ -375,7 +375,7 @@ public class Faucet : FaucetAPI
                 .filter!(kv => kv.value.output.value >= minInputValuePerOutput)
                 .take(uniform(2, config.tx_generator.merge_threshold, rndGen));
             if (utxo_rng.empty)
-                logInfo("Waiting for unspent utxo");
+                logInfo("\tWaiting for unspent utxo");
```

Comment:
I would recommend against assuming it goes to `stdout`.

Reply:
There were existing `logInfo` with tabs so this was to keep it aligned. Shall I add a commit to remove those first?

Reply:
Ah, if it follows existing practice, let's go with this for now.